### DISCUSSION TITLE
What does the "move your cat" extension?

### DISCUSSION THREAD

**Discussion by NoobTracker** [ScratchAddons/ScratchAddons#149] [Help]
I really don't understand what that extension does.

**Comment by WorldLanguages** [comment]
@apple502j made it to test the prototype pollution API he created. It's not really useful at the moment.

**Comment by NoobTracker** [comment]
I still don't really understand it but okay.

**Comment by WorldLanguages** [comment]
Yeah, it's a very technical API that is useful for a few addons that enhance the Scratch editor.

**Comment by Explosion-Scratch** [comment]
> Yeah, it's a very technical API that is useful for a few addons that enhance the Scratch editor.

So should it be removed?
